samples/linear_layout/main.go

```diff
@@ -5,15 +5,26 @@
 package main
 
 import (
+	"flag"
 	"github.com/google/gxui"
 	"github.com/google/gxui/drivers/gl"
 	"github.com/google/gxui/math"
-	"github.com/google/gxui/samples/flags"
 	"github.com/google/gxui/themes/dark"
+	"github.com/google/gxui/themes/light"
+)
+
+var (
+	flagTheme          = flag.String("theme", "dark", "Theme to use {dark|light}.")
```

Comment:
Possibly, but then would need to duplicate the code that picks the theme based on the flag value. That could probably be generalized somewhere inside gxui itself so you just do something like

theme := CreateNamedTheme(flagTheme);

and it does the right thing? But that sounds like a separate CL.